routes/runtime/github.rb

```diff
@@ -6,13 +6,6 @@ class Clover
       fail CloverError.new(400, "InvalidRequest", "invalid JWT format or claim in Authorization header")
     end
 
-    begin
-      repository.setup_blob_storage unless repository.access_key
-    rescue Excon::Error::HTTPStatus => ex
-      Clog.emit("Unable to setup blob storage") { {failed_blob_storage_setup: {ubid: runner.ubid, repository_ubid: repository.ubid, response: ex.response.body}} }
-      fail CloverError.new(400, "InvalidRequest", "unable to setup blob storage")
-    end
-
     # getCacheEntry
     r.get "cache" do
       keys, version = typecast_params.nonempty_str!(%w[keys version])
```

Comment:
Great finding!
Should we add `next 204 unless repository.access_key` here as early exit instead of running all queries?